src/version.c

```diff
@@ -0,0 +1,14 @@
+#include "version.h"
+
+#define TOSTRING(x) #x
+#define EXPAND(x) TOSTRING(x)
+
+const char* picorb_version(void) {
```

Comment:
```suggestion
const char *
picorb_version(void) {
```

Function name should locate at the beginning of the line by this project's convention

Reply:
And `version.h` should contain the prototype of the function